plugins.json

```diff
@@ -6,49 +6,56 @@
         "pip_url": "aiida-core",
         "plugin_info": "https://raw.githubusercontent.com/aiidateam/aiida-core/master/setup.json",
         "code_home": "https://github.com/aiidateam/aiida-core",
-        "documentation_url": "https://aiida-core.readthedocs.io/"
+        "documentation_url": "https://aiida-core.readthedocs.io/",
+        "pypi_name": "aiida-core"
```

Comment:
Correct me if I'm wrong but I don't think this additional field is necessary.

As described in the README, the `pip_url` will contain either the package name or a URL to the repo.

One can argue about whether that is the best solution, but it has been in place since a while, so I suggest you simply get the package name from there.

Reply:
Yep sure, will just have to add a bit of logic that decides if it is a pypi name (and so we can source metadata from there) or an external url (so we can only the `plugin_info` url for metadata)

Reply:
Ok fixed in [`03f09bd` (#201)](https://github.com/aiidateam/aiida-registry/pull/201/commits/03f09bd1e7da8eb0756c9591261822447ac64639)